### ISSUE TITLE
Use a better color picker

### ISSUE THREAD

**Issue by KOLANICH** [MCMrARM/revolution-irc#94]
It may be good to add a color picker allowing  to choose any color. There are libraries for that.

**Comment by MCMrARM** [comment]
Which one? The one for primary/secondary colors or the one for notifications?
The one for the primary/secondary allows you to chose from any material colors, you should not use colors outside the palette and I'm *definitely* not going to replace that one, but I could add a RGB color picker.
If you mean the color picker for text, yea, that one could be better.

**Comment by KOLANICH** [comment]
I meant all the color pickers in the app. It's completely OK not to replace but to add.

**Comment by MCMrARM** [comment]
There are two color picker implementations used within the app.

**Comment by KOLANICH** [comment]
I guess there should be a one universal color picker with 2 modes: one is presets, another one is "any color". And a button to switch modes.

**Comment by kurahaupo** [comment]
How do I select actual black #000000 rather than just dark grey, for the background?

When I first started using RevolutionIRC (a month or so ago) I had trouble with unexpected elements of the UI changing colour when I was trying to set some other elements' colours. Each time that happened, I had to reset the colours and start from scratch. Being to save a personal theme would allow us to experiment.
